src/Android/Xamarin.Android/Samples/Layers/ChangeStretchRenderer/ChangeStretchRenderer.cs

```diff
@@ -0,0 +1,345 @@
+
+            // Initialize the app
+            Initialize();
+        }
+
+        private void CreateLayout()
+        {
+            // This section creates the UI elements and adds them to the layout view of the GUI
+
+            // Create a stack layout
+            LinearLayout layout = new LinearLayout(this) { Orientation = Orientation.Vertical };
+
+            // Create a list of stretch renderer choices the user can choose from
+            _RendererTypes = new ListView(this);
+            _RendererTypes.ItemClick += RendererTypes_SelectionChanged;
+            layout.AddView(_RendererTypes);
+
+            // ------------
+
+            // Create a sub layout to organize the label and 1st parameter on a single horizontal line
+            var subLayout1 = new LinearLayout(this) { Orientation = Orientation.Horizontal };
+
+            // Create label that displays the 1st parameter used by the stretch renderer
+            _Label_Parameter1 = new TextView(this);
+            _Label_Parameter1.Text = "Minimum value (0 - 255):";
+            subLayout1.AddView(_Label_Parameter1);
+
+            // Create text field for 1st parameter used by the stretch renderer that the user can modify 
+            _Input_Parameter1 = new EditText(this);
+            _Input_Parameter1.Text = "10";
+            subLayout1.AddView(_Input_Parameter1);
+
+            // Add the sub layout to the main layout
+            layout.AddView(subLayout1);
+
+            // ------------
+
+            // Create a sub layout to organize the label and 2nd parameter on a single horizontal line
+            var subLayout2 = new LinearLayout(this) { Orientation = Orientation.Horizontal };
+
+            // Create label that displays the 2nd parameter used by the stretch renderer
+            _Label_Parameter2 = new TextView(this);
+            _Label_Parameter2.Text = "Maximum value (0 - 255):";
+            subLayout2.AddView(_Label_Parameter2);
+
+            // Create text field for 2nd parameter used by the stretch renderer that the user can modify 
+            _Input_Parameter2 = new EditText(this);
+            _Input_Parameter2.Text = "150";
+            subLayout2.AddView(_Input_Parameter2);
+
+            // Add the sub layout to the main layout
+            layout.AddView(subLayout2);
+
+            // ------------
+
+            // Create button to change stretch renderer of the raster
+            _UpdateRenderer = new Button(this);
+            _UpdateRenderer.Text = "UpdateRenderer";
+            // Hook to touch/click event of the button
+            _UpdateRenderer.Click += OnUpdateRendererClicked;
+            layout.AddView(_UpdateRenderer);
+
+            // Create the map view and add it to the main layout
+            _myMapView = new MapView(this);
+            layout.AddView(_myMapView);
+
+            // Set the layout as the sample view
+            SetContentView(layout);
+        }
+
+        private async void Initialize()
```

Comment:
On Android and iOS we can make this `async Task`, and then make the calling method (`OnCreate` in this case) async as well. This hasn't been done in the past, but I believe it is the more correct way to do things. WPF/UWP/Forms will need to have some more work done to make this possible, so I don't recommend changes on those platforms. 

Alternatively, you could ignore the comment and the issue will be fixed consistently for all samples when it is addressed.